src/NeptuneSchema.js

```diff
@@ -318,8 +346,10 @@ async function getNeptuneSchema(quiet) {
     
     try {
         await getAWSCredentials();
-    } catch (error) {        
-        consoleOut("There are no AWS credetials configured. \nGetting the schema from an Amazon Neptune database with IAM authentication works only with AWS credentials.");
+    } catch (error) {
+        msg = "There are no AWS credetials configured. \nGetting the schema from an Amazon Neptune database with IAM authentication works only with AWS credentials.";        
```

Comment:
Can you please fix the previous spelling error for 'credentials' since you're modifying this line anyways?

Reply:
Fixed!